<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistWindow</class>
 <widget class="QDockWidget" name="PlaylistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>491</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>P&amp;laylist</string>
  </property>
  <widget class="QWidget" name="contents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="tabStack"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="searchHost" native="true">
      <layout class="QHBoxLayout" name="searchLayout" stretch="1,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchField">
         <property name="placeholderText">
          <string>Search</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="visibleToQueue">
         <property name="toolTip">
          <string>Visible Items to Quick Queue</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/controls/playlist/quickqueue.png</normaloff>:/images/controls/playlist/quickqueue.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="newTab">
        <property name="toolTip">
         <string>New Tab</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/controls/playlist/newtab.png</normaloff>:/images/controls/playlist/newtab.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeTab">
        <property name="toolTip">
         <string>Close Tab</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/controls/playlist/closetab.png</normaloff>:/images/controls/playlist/closetab.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="duplicateTab">
        <property name="toolTip">
         <string>Duplicate Tab</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/controls/playlist/duplicatetab.png</normaloff>:/images/controls/playlist/duplicatetab.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importList">
        <property name="toolTip">
         <string>Import Playlist</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/controls/playlist/import.png</normaloff>:/images/controls/playlist/import.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportList">
        <property name="toolTip">
         <string>Export Playlist</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/controls/playlist/export.png</normaloff>:/images/controls/playlist/export.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tabStack</tabstop>
  <tabstop>searchField</tabstop>
  <tabstop>visibleToQueue</tabstop>
  <tabstop>newTab</tabstop>
  <tabstop>closeTab</tabstop>
  <tabstop>duplicateTab</tabstop>
  <tabstop>importList</tabstop>
  <tabstop>exportList</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
